docs/content/preview/api/ysql/syntax_resources/ysql_grammar.ebnf

```diff
@@ -1237,7 +1237,7 @@ expression_list = expression { ',' expression } ;
 (* For the invocation of a window function, or an aggregate function, in conjunction with the OVER keyword.*)
 (* And for the invocation of an aggregate function, in conjunction with the GROUP BY clause.*)
 
-select_start = 'SELECT' [ 'ALL' | 'DISTINCT' [ 'ON' ( '(' expression { ',' expression } ')' ) ] ]
+select_start = 'SELECT' [ 'ALL' | 'DISTINCT' [ 'ON' ( '(' expression { ',' expression } ')' ) ] ] \
```

Comment:
this should commit as a separate change beforehand or be documented

Reply:
This is removed.